src/Init/Lean/Parser/Parser.lean

```diff
@@ -234,6 +225,15 @@ def toOptional : FirstTokens → FirstTokens
 | tokens tks => optTokens tks
 | tks        => tks
 
+def merge : FirstTokens → FirstTokens → FirstTokens
+| epsilon,      tks          => toOptional tks
+| tks,          epsilon      => toOptional tks
```

Comment:
The `toOptional` makes sense. Is this an independent fix, or was it creating problems for this PR?

Reply:
Without it, existing syntax broke when introducing the new antiquotations.